frontend/src/Components/StoreSearch/StoreSearch.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import StoreCard from "./StoreCard";
+import {Container, Col, Row, Jumbotron, Button, ButtonGroup, Card, InputGroup, FormControl} from "react-bootstrap";
+import './StoreSearch.css'
+
+const stores = [
+    {
+        name: "Super Market",
+        address: "123 E Green St. Champaign, IL",
+        hours: [9, 18],
+        distance: 6
+    },
+    {
+        name: "Local Grocer",
+        address: "456 Testing Ave. Boston, MA",
+        hours: [8, 16],
+        distance: 11
+    }
+    ]
+
+const getLocation = () => {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(processLocation, locationError);
+    } else {
+        console.error("Geolocation is not supported by this browser.");
+    }
+};
+
+const processLocation = (pos) => {
+    console.log([pos.coords.latitude, pos.coords.longitude]);
+};
+
+const locationError = (error) => {
+    switch(error.code) {
+        case error.PERMISSION_DENIED:
+            console.error("User denied the request for Geolocation.");
+            break;
+        case error.POSITION_UNAVAILABLE:
+            console.error("Location information is unavailable.");
+            break;
+        case error.TIMEOUT:
+            console.error("The request to get user location timed out.");
+            break;
+        case error.UNKNOWN_ERROR:
```

Comment:
I'm not too familiar with the different types of possible error codes. When is it possible for UNKNOWN_ERROR to occur in this context?